pkg/lib/k8s/pod.go

```diff
@@ -59,6 +61,9 @@ var _killStatuses = map[int32]bool{
 	129: true, // SIGHUP
 }
 
+// https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/images/types.go#L27
```

Comment:
Use the Kubernetes version used in Cortex here https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/images/types.go#L27?